wolfssl/openssl/asn1.h

```diff
@@ -92,6 +92,17 @@
 #define ASN1_STRING_FLAG_MSTRING         0x040
 #define ASN1_STRING_FLAG_EMBED           0x080
 
+/* X.509 PKI size limits from RFC2459 (appendix A) */
+/* internally our limit is CTC_NAME_SIZE (64) - overriden with WC_CTC_NAME_SIZE */
+#define ub_name                    CTC_NAME_SIZE /* 32768 */
```

Comment:
What are the numbers in the comments at the end of the lines?

Reply:
Those are the RFC defined max limits (commented for reference), but each field is limited by CTC_NAME_SIZE, which can be build-time overridden.